demisto_sdk/commands/lint/helpers.py

```diff
@@ -189,7 +190,7 @@ def add_typing_module(lint_files: List[Path], python_version: float):
     back_lint_files: List[Path] = []
     try:
         # Add typing import if needed to python version 2 packages
-        if python_version < 3:
+        if float(python_version) < 3:
```

Comment:
What happens when the version is `3.10.1`? or do we not provide the revision number?

Reply:
I updated this method too